vaadin-notification-flow-parent/vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/Notification.java

```diff
@@ -219,7 +222,12 @@ private void initBaseElementsAndListeners() {
     private void removeAutoAdded() {
         if (autoAddedToTheUi && !isOpened()) {
             autoAddedToTheUi = false;
-            getElement().removeFromParent();
+            try {
```

Comment:
Instead of try-catch, do the following check
`if (getElement().getParentNode().getChildren().anyMatch(n -> n.equals(getElement()))) {`